docs/getting_started/user-guide.rst

```diff
@@ -474,14 +489,14 @@ Container-specific scripts you'll want to include in the container.yaml, describ
 
 Since different container technologies might expose different environment variables (e.g., ``SINGULARITY_OPTS`` vs ``PODMAN_OPTS``)
 they are organized above based on the container technology. If you want to customize the wrapper script, simply replace the relative paths
-above (e.g., ``singularity.sh`` with an absolute path to a file that will be used instead. For global alias scripts such as these, 
+above (e.g., ``singularity.sh``) with an absolute path to a file that will be used instead. For global alias scripts such as these, 
 Singularity HPC will look for:
 
 1. An absolute path first, if found is used first.
 2. Then a script name in the shpc/main/wrappers directory
 
 Here is an example of using wrapper scripts for the "python" container, which doesn't have container specific wrappers. What you see
-is the one entrypoint, "python" being placed in a bin that the module will see instead of defining the alias.
+is the one entrypoint, "python", being placed in a "bin" subdirectory that the module will see instead of defining the alias.
```

Comment:
If you don't want the comma on the inside, then let's make it a `command`
```suggestion
is the one entrypoint, `python`, being placed in a "bin" subdirectory that the module will see instead of defining the alias.
```

Reply:
sorry @vsoch , my glitch on this one, got confused and missed this